ovos_config/__main__.py

```
@@ -39,30 +39,37 @@ def dictDepth(dic: dict, level: int = 1) -> int:
-             key: str,
-             full_path: bool = False,
-             path: Tuple = (),
-             found: bool = False):
+    return all(key_part.lower() in path_part.lower() for (key_part, path_part) in zip(key, path[-len(key):]))
+
+def walkDict_(dic: dict,
+              key: Tuple,
+              key_absolute: bool,
+              only_endpoints: bool = False,
+              path: Tuple = ()):
     for k in dic.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)